src/de/mod.rs

```diff
@@ -1106,27 +1097,24 @@ mod tests {
             assert_eq!(
                 de.read,
                 vec![
-                    Start(BytesStart::borrowed_name(b"inner")),
+                    Start(QName(b"inner").into()),
                     Text(BytesText::from_escaped_str("text")),
```

Comment:
I'm mildly negative on this because it makes using the API less consistent.  That's not to say that the implementation shouldn't be there, because it makes sense to have the implementation, but in practice it flows awkwardly.

Not a blocker, just FWIW